src/main/kotlin/no/nav/familie/ef/søknad/integration/SøknadClient.kt

```diff
@@ -66,9 +67,9 @@ class SøknadClient(private val config: MottakConfig,
     }
 
     fun hentSøknaderMedDokumentasjonsbehov(personIdent: String): List<SøknadMedDokumentasjonsbehovDto> {
-        return postForEntity(config().hentSøknaderMedDokumentasjonsbehovUri,
-                             PersonIdent(personIdent),
-                             HttpHeaders().medContentTypeJsonUTF8())
+        return filtrerVekkEldreDokumentasjonsbehov(postForEntity(config().hentSøknaderMedDokumentasjonsbehovUri,
```

Comment:
Her tror jeg det hadde vært nyttig med en variabel 
`val søknaderMedDokumentasjonsbehov = postForEntity(...)`
 for å lett se at vi gjør et kall og henter data - og deretter filtrerer dette. 
Da går det kanskje også an å gjøre `return søknaderMedDokumentasjonsbehov.filter(søknaderDeSisteFireUkene)` dersom det er lettere "flytmessig" enn `filtrerVekk` - men det siste er bare en liten detalj.